packages/react/src/components/drop/DropPod.tsx

```diff
@@ -213,9 +213,12 @@ const RDropPod: FunctionComponent<IRDropPodProps> = ({
             // Restrict the max-width to the inner width of the closest relatively positioned ancestor
             if (relativeParent) {
                 const {paddingLeft, paddingRight} = getComputedStyle(relativeParent) ?? {};
```

Comment:
Maybe we should set paddingLeft and paddingRight to 0 by default?

Reply:
I tried it and I would get a maxWidth of 0 in the admin because the parent offset is 0 in my bug which is not nice 😆 🙈